app/views/admin/home.erb

```diff
@@ -20,7 +20,7 @@
         <td><%= s.overall_grade %></td>
 
         <!-- You will need to add the overall adjusted grade yourself: -->
-        <td></td>
+        <td><%= s.adjusted_grade_for_assignment(@assignment) %></td>
```

Comment:
There should be some method in the `Student` class that provides an _overall_ adjusted grade. This is not it.